Example/URBNDataSource/ViewControllers/Swift CollectionView/URBNSwiftCollectionViewController.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  URBNSwiftCollectionViewController.swift
+//  URBNDataSource
+//
+//  Created by Dustin Bergman on 12/14/15.
+//  Copyright © 2015 Joe. All rights reserved.
+//
+
+import UIKit
+
+class URBNSwiftCollectionViewController: UICollectionViewController {
+
+    var adapter: URBNArrayDataSourceAdapter?
```

Comment:
Why not go ahead and initialize the adapter here.  Then you could get rid of all the dirty "?"